### DISCUSSION TITLE
Is there an passportJS example available

### DISCUSSION THREAD

**Discussion by wortkotze** [supabase/supabase#571] [Questions]
Hi, I try to move an application from nextjs over to supabase. We are using for our BFF passportjs with jwt. is there any example or passport js strategy that we can use for the auth service?

thx for the help
Stephan

**Comment by kiwicopple** [comment]
Hi @wortkotze Unfortunately not yet... we would love some contributions to our examples so if you end up building this and you're happy to share some code that would be greatly appreciated 🙏
